packages/shared/src/components/modals/common.tsx

```diff
@@ -136,6 +136,7 @@ export const modals = {
   [LazyModal.MarketingCta]: MarketingCtaModal,
   [LazyModal.UserSettings]: UserSettingsModal,
   [LazyModal.Share]: ShareModal,
+  [LazyModal.PublicSquadSubmitForReview]: SquadTourModal, // TODO: placeholder, fix this when working on https://dailydotdev.atlassian.net/browse/MI-339
```

Comment:
Just note it in a task as well 🙏

Reply:
[done](https://dailydotdev.atlassian.net/browse/MI-339?focusedCommentId=11683)